pkg/occlient/occlient.go

```diff
@@ -1052,36 +1052,59 @@ func (c *Client) FollowBuildLog(buildName string) error {
 	return nil
 }
 
-// Delete calls oc delete
-// kind is always required (can be set to 'all')
-// name can be omitted if labels are set, in that case set name to ''
-// if you want to delete object just by its name set labels to nil
-func (c *Client) Delete(kind string, name string, labels map[string]string) (string, error) {
+// Delete takes labels as a input and based on it, deletes respective resource
+func (c *Client) Delete(labels map[string]string) error {
+	// convert labels to selector
+	selector := util.ConvertLabelsToSelector(labels)
 
-	args := []string{
-		"delete",
-		kind,
+	var errorList []string
+	// Delete DeploymentConfig
```

Comment:
It would be good to add some debug messages to see what is happening and progress of deletion.
Like selector that is being used to delete objects, and what type of objects are being deleted.

Reply:
added :)